internal/engine/templates/openapi/tiles.go.json

```diff
@@ -23,7 +23,7 @@
           "Tiling Schemes"
         ],
         "summary": "Retrieve the list of available tiling schemes (tile matrix sets)",
-        "operationId": "getTileMatrixSetsList",
+        "operationId": "getTileMatrixSetsList.dataset.vector.getTileSetsList",
```

Comment:
moet die prefix er nog wel voor, getTileMatrixSetsList? Kan dat niet weg.

Reply:
De spec zegt "The `paths` defined in the OpenAPI definition SHALL have an *operationId* value ending with the relevant dot-separated suffix corresponding to the resource as specified in [Table 11](https://docs.ogc.org/is/20-057/20-057.html#openapi-operation-id-suffixes)." - die "*ending with*" lees ik als dat er dus ook nog iets voor moet staan.

Reply:
Helemaal weghalen maakt dat de operationIds niet meer uniek zijn, maar kan hier wel `tileMatrixSets.dataset.vector.getTileSetsList` van maken, dat is duidelijker en heeft niet de dubbele `get` erin.